File: C1.UWP.FlexChart/VB/StockChart/StockChart/Converters/SeriesVisibility2BooleanConverter.vb
Public Class SeriesVisibility2BooleanConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Dim v As C1.Chart.SeriesVisibility = DirectCast(value, C1.Chart.SeriesVisibility)
        Return v = C1.Chart.SeriesVisibility.Visible
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Dim b As Boolean = System.Convert.ToBoolean(value)
        Return If(b, C1.Chart.SeriesVisibility.Visible, C1.Chart.SeriesVisibility.Hidden)
    End Function
End Class

File: C1.UWP.FlexChart/CS/StockAnalysis/StockAnalysis/Converter/SeriesVisibility2BooleanConverter.cs
using System;
using System.Globalization;
using Windows.UI.Xaml.Data;

namespace StockAnalysis.Converter
{
    public class SeriesVisibility2BooleanConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            C1.Chart.SeriesVisibility visibility = (C1.Chart.SeriesVisibility)value;
            return visibility != C1.Chart.SeriesVisibility.Visible;
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            throw new NotImplementedException();
        }
    }
}
